addons/web/static/src/js/control_panel/search_bar.js

```diff
@@ -231,6 +232,25 @@ odoo.define('web.SearchBar', function (require) {
             }
         }
 
+        /**
+         * @private
+         * @returns {Object[]}
+         */
+        _getFacets() {
+            const facets = this.model.get("facets").map((facet) => {
+                facet.separator = facet.type === 'groupBy' ? ">" : this.env._t("or");
+                const icon = FACET_ICONS[facet.type];
+                if (icon) {
+                    delete facet.title;
```

Comment:
hum, I would have preferred to avoid that

Reply:
and why is the all function here and not in ActionModel?

Reply:
I wanted to put as much as possible here rather than in the model; this is rendering logic so I don't think it should be put higher up. For the title part I will change the template and remove this.

Reply:
Done